app/controllers/new-signup.js

```diff
@@ -64,12 +79,28 @@ export default class NewSignUpController extends Controller {
     else this.isButtonDisabled = true;
   }
 
+  @action handleCheckboxInputChange(key, value) {
+    set(this.signupDetails.roles, key, value);
```

Comment:
What is this set for?

Reply:
set key and value inside object signupDetails.roles